.dockerignore

```diff
@@ -0,0 +1,6 @@
+.env
+.venv
+.dockerignore
+Dockerfile
+**/__pycache__
+
```

Comment:
Nice. 

There are other things you could include (like .git or .vagrant) but they aren't necessary since your COPY commands in the Dockerfile are quite targeted.